spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  before(:all) do
+    Rails.application.load_seed
+  end
+
+  before { get user_posts_path(user_id: User.first.id) }
+  describe 'GET /posts' do
+    before { user_posts_path(1) }
+
+    it 'returns correct status code' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'Should render index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'Should have text Index Posts' do
+      expect(response.body).to include('Here is a list of posts for a given user')
+    end
+  end
+
+  describe 'GET /posts/:id' do
+    before { get user_post_path(1, 1) }
+    it 'returns correct status code' do
+      get user_post_path(User.first.id, id: Post.first.id)
+      expect(response).to have_http_status(200)
+    end
+
+    it 'Should render show template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'Should have text Show Post' do
+      expect(response.body).to include('Here is a specific post')
+    end
+  end
+end
```

Comment:
- [ ] All the tests fail after running the `rspec spec/requests` command. Please ensure that your tests pass successfully in the terminal.
- [ ] It would be nicer to add some steps to your Readme on how the user can run the tests locally.

<img width="457" alt="image" src="https://user-images.githubusercontent.com/77190203/180219428-04dcb821-d8ab-441a-8fa0-d0ced6ee96bc.png">